src/pages/new-chat/new-chat.ts

```diff
@@ -9,22 +12,30 @@ import { User } from 'api/models/whatsapp-models';
   templateUrl: 'new-chat.html'
 })
 export class NewChatComponent implements OnInit {
-  users;
+  searchPattern = '';
   senderId: string;
+  users: Observable<User[]>;
 
   constructor(
-    public navCtrl: NavController,
-    public viewCtrl: ViewController,
-    public alertCtrl: AlertController
+    private navCtrl: NavController,
+    private viewCtrl: ViewController,
+    private alertCtrl: AlertController
   ) {
     this.senderId = Meteor.userId();
```

Comment:
It's been already done at the entry script over here: https://github.com/DAB0mB/Ionic2CLI-Meteor-WhatsApp/blob/47c805c8619936cfb4cefd90bee93cf7e8f6c462/src/app/main.ts#L11
Ionic doesn't use any routes, it uses the navigation controller instead. So whatever page is loaded, it must go through the entry script first, which means the use case is already covered.